packages/suite/src/components/suite/Labeling/components/AddressLabeling.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import * as accountUtils from '@wallet-utils/accountUtils';
```

Comment:
Import jus that one function that we use here

Reply:
done